scripts/render-notebooks.R

```diff
@@ -59,6 +59,9 @@ opt <- parse_args(OptionParser(option_list = option_list))
 # Get working directory
 base_dir <- getwd()
 
+# Normalize file path
+opt$bib_file <- normalizePath(opt$bib_file)
+
```

Comment:
These lines are not necessary anymore, so I don't know how they are sneaking back in... Suggesting deletion.
```suggestion
```